install_config/aggregate_logging.adoc

```diff
@@ -741,37 +741,45 @@ $ oc get route  --all-namespaces --selector logging-infra=support
 ----
 ====
 
-== External Elasticsearch instance with Fluentd
+[[sending-logs-to-an-external-elasticsearch-instance]]
+== Sending Logs to an External Elasticsearch instance
 
-It is possible to configure the Fluentd pod created with aggregated logging to
-connect to an externally hosted Elasticsearch instance.
+Fluentd sends logs to the value of the `ES_HOST`, `ES_PORT`, `OPS_HOST`, and
+`OPS_PORT` environment variables of the Elasticsearch deployment configuration.
+The application logs are directed to the `ES_HOST` destination, and operations
+logs to `OPS_HOST`.
 
-Fluentd knows where to send its logs to based on the `ES_HOST`, `ES_PORT`,
-`OPS_HOST` and `OPS_PORT` environment variables.  If you have an external
-Elasticsearch instance that will contain both application and operations logs,
-ensure that `ES_HOST` and `OPS_HOST` are the same and that `ES_PORT` and
-`OPS_PORT` are also the same.  Fluentd is configured to send its application logs
-to the `ES_HOST` destination and all of its operations logs to `OPS_HOST`.
+To direct logs to a specific Elasticsearch instance, edit the deployment
+configuration and replace the value of the above variables to the desired
```

Comment:
Hmm, "replace  ... to the desired instance" is new usage to me.  How about "replace ... with the desired instance"?